mora/service/common.py

```diff
@@ -459,3 +459,15 @@ def get_valid_to(obj, fallback=None):
         elif valid_to:
             return util.from_iso_time(valid_to)
     return util.positive_infinity
+
+
+def is_reg_valid(reg, gyldighed_key):
+    """
+    Check if a given registration is valid
+
+    :param reg: A registration object
+    :param gyldighed_key: A key for where to find the gyldighed on the object
+    :return: True if registration 'gyldighed' is 'Aktiv'
+    """
+    return bool(get_obj_value(reg, ('tilstande', gyldighed_key),
+                              lambda x: x.get('gyldighed') == 'Aktiv'))
```

Comment:
Couldn't you simplify this code a bit by iterating over the states, and check if any are active?

Reply:
That is essentially what's happening already. But it could probably be simplified a bit.

Reply:
I mean that you could iterate over all states rather than specify `gyldighed_key` — that seems a bit cleaner to me?

Reply:
That's a good point